editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1377,19 +1377,21 @@ AnimationPlayerEditor::AnimationPlayerEditor(EditorNode *p_editor) {
 	name_dialog->set_title(TTR("Create New Animation"));
 	name_dialog->set_hide_on_ok(false);
 	add_child(name_dialog);
-	name = memnew(LineEdit);
-	name_dialog->add_child(name);
-	name->set_position(Point2(18, 30));
-	name->set_anchor_and_margin(MARGIN_RIGHT, ANCHOR_END, 10);
-	name_dialog->register_text_enter(name);
+	VBoxContainer *vb = memnew(VBoxContainer);
+	name_dialog->add_child(vb);
 
 	l = memnew(Label);
 	l->set_text(TTR("Animation Name:"));
 	l->set_position(Point2(10, 10));
-
-	name_dialog->add_child(l);
+	vb->add_child(l);
 	name_title = l;
 
+	name = memnew(LineEdit);
+	vb->add_child(name);
+	name->set_position(Point2(18, 30));
```

Comment:
I think you shouldn't be doing this -- the container is supposed to manage the positions of its children.